### PR TITLE
fix and enable more unittests

### PR DESCRIPTION
**Pull request by Shreeshrii**

### CODE REVIEW

File: unittest/lang_model_test.cc [hunk 1]
```diff
@@ -111,4 +214,4 @@ TEST(LangModelTest, AddACharacter) {
 }
 
 }  // namespace
-}  // namespace tesseract
+}  // namespace tesseract
```
Comment (stweil) [inline unittest/lang_model_test.cc]: Here you accidentally removed the terminating linefeed.

Comment (Shreeshrii) [reply unittest/lang_model_test.cc]: Fixed via 5e90a61eb4e27d00d70a8385369bd969dd786df4
Thanks!

File: src/training/tesstrain_utils.sh [hunk 2]
```diff
@@ -246,7 +250,7 @@ generate_font_image() {
 
     local common_args="--fontconfig_tmpdir=${FONT_CONFIG_CACHE}"
     common_args+=" --fonts_dir=${FONTS_DIR} --strip_unrenderable_words"
-    common_args+=" --leading=${LEADING}"
+    common_args+=" --leading=${LEADING} --xsize=${X_SIZE}"
```
Comment (stweil) [inline src/training/tesstrain_utils.sh]: The argument is called `--xsize` here and also in the subject of this commit, but the commit implements it as `--x_size`. I think this is confusing and suggest to always use `--xsize`.

Comment (Shreeshrii) [reply src/training/tesstrain_utils.sh]: Thanks. Fixed via 94b8988b8cca3812137933db00750bd6e2e84e32

**Pull request merged**